tailwarden/komiser
# Restore OVH provider

## Problem

#912 

## How to Test

- Try to configure your OVH account with Komiser like 
```
[[ovh]]
  name="Test"
  endpoint="ovh-eu"
  application_key="my_app_key"
  application_secret="my_application_secret"
  my_consumer_key="my_consumer_key"
```

- See if resources are correctly being fertched

## Notes

- Haven't tested it yet

## Checklist
